static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -233,10 +232,6 @@ function EventsTable({
       if (field === 'id' || field === 'trace') {
         const isIssue = !!issueId;
         let target: LocationDescriptor = {};
-        const locationWithTab = {
```

Comment:
We no longer push tabs to the url for insights, we have sub-routes. This is safe.